versioning/BRM/slavecomm.cpp

```diff
@@ -45,6 +45,8 @@
 
 #include "installdir.h"
 
+#define BRM_VERBOSE
```

Comment:
I don't think this macros will activate logging on its own but it is better to remove it.

Reply:
Yes, thanks.